sql/mob_droplist.sql

```diff
@@ -26854,6 +26856,29 @@ INSERT INTO `mob_droplist` VALUES (3254,0,0,1000,880,@VCOMMON); -- Bone Chip (Ve
 INSERT INTO `mob_droplist` VALUES (3254,0,0,1000,4824,@RARE);   -- Scroll Of Gravity (Rare, 5%)
 INSERT INTO `mob_droplist` VALUES (3254,2,0,1000,880,0);        -- Bone Chip (Steal)
 
+-- ZoneiD: 25 - Orcish Footsoldiers
```

Comment:
Rest of the file is `ZoneID`, not `ZoneiD`. This was all re-generated with  tooling, so make sure it all adheres to the existing style

Reply:
Fair enough - I meant to change that but obviously forgot to do it - much like when testing with the @always at one point. Updated title and squashed. Thanks!